app/controllers/users.js

```diff
@@ -218,7 +218,7 @@ const addDonation = (req, res) => {
         _id: req.user._id
       })
         .exec((err, user) => {
-        // Confirm that this object hasn't already been entered
+          // Confirm that this object hasn't already been entered
           let duplicate = false;
           for (let i = 0; i < user.donations.length; i += 1) {
```

Comment:
We should totally avoid using iterators as much as we can .Perfer using array functions